doc/newfeat.texi

```diff
@@ -4,6 +4,13 @@
 Changes and New Features in development version:
 @itemize @bullet
 
+@item @ESS{[R]}: Fontification of roxygen @code{@@param} keywords now
+supports comma-separated parameters.
+
+@item @ESS{[R]}: Function-like keywords such as @code{if} or @code{stop}
+are no longer fontified as keyword if not followed by an opening
```

Comment:
Why just `if`, `stop` and not all keywords?

Reply:
If your taks is to fix spurious fontification,  then I see no reason at all to distinguish between "function like" keywords and other keywords. 

What is "function-like" keywords in the first place?

Reply:
A keyword that precedes an opening parenthesis. I've been greatly annoyed by spurious fontifications of `if`, `message` etc. Now they are only fontified if an opening parenthesis follows.

> I see no reason at all to distinguish between "function like" keywords and other keywords.

What do you mean? The goal is to keep fontifying `next`, `break` and other non-function-like keywords that could cause a jump.

Reply:
With `next`, `break` no chance, but with `while`, `for`, `while`, `switch` etc. it's possible.

Reply:
What do you mean?

Reply:
What is not clear more concretely? while, switch, stop etc. are function like, just like `if` and `for`. So all I am saying is that your fix should apply to those as well, not just to `if` and `for`.

Reply:
> So all I am saying is that your fix should apply to those as well, not just to if and for.

That is the case, it's now even covered by unit tests.

Reply:
> That is the case

I see. Should have been more careful with reading your PR :/, sorry.